testing/test_assertion.py

```diff
@@ -910,7 +909,9 @@ class SimpleDataObject:
 
         lines = callequal(left, right)
         assert lines is not None
-        assert "field_d: 'a' != 'b'" in lines
+        assert (
+            "    field_d: 'a' != 'b'" in lines
+        )  # indentation in output because of nested object structure
```

Comment:
```suggestion
        # Indented because of nested object structure.
        assert "    field_d: 'a' != 'b'" in lines
```

Reply:
I guess, the formatter proposed to put it into the brackets. But I will check